docs/quickstart.md

```diff
@@ -0,0 +1,98 @@
+# Quickstart
+
+This is a quickstart guide for getting Cluster API Operator up and running on your Kubernetes cluster.
+
+For more detailed information, please refer to the full documentation.
+
+## Prerequisites
+
+- [Running Kubernetes cluster](https://cluster-api.sigs.k8s.io/user/quick-start#install-andor-configure-a-kubernetes-cluster).
+- [kubectl](https://kubernetes.io/docs/tasks/tools/install-kubectl/) for interacting with the management cluster.
+- [Helm](https://helm.sh/docs/intro/install/) for installing operator on the cluster (optional).
+
+## Install and configure Cluster API Operator
+
+### Configuring credential for cloud providers
+
+Instead of using environment variables as clusterctl does, Cluster API Operator uses Kubernetes secrets to store credentials for cloud providers. Refer to [provider documentation](https://cluster-api.sigs.k8s.io/user/quick-start#initialization-for-common-providers) on which credentials are required.
+
+This example uses AWS provider, but the same approach can be used for other providers.
+
+```bash
+export CREDENTIALS_SECRET_NAME="credentials-secret"
+export CREDENTIALS_SECRET_NAMESPACE="default"
+
+kubectl create secret generic "${CREDENTIALS_SECRET_NAME}" --from-literal=AWS_B64ENCODED_CREDENTIALS="${AWS_B64ENCODED_CREDENTIALS}" --namespace "${CREDENTIALS_SECRET_NAMESPACE}"
+```
+
+### Installing Cluster API Operator
+
+Add helm repository:
+
+```bash
+helm repo add capi-operator https://kubernetes-sigs.github.io/cluster-api-operator
+helm repo update
+```
+
+Deploy Cluster API components with docker provider with a single command during operator installation:
+
+```bash
+helm install capi-operator capi-operator/cluster-api-operator --create-namespace -n capi-operator-system --set infrastructure=docker --set cert-manager.enabled=true --set configSecret.name=${CREDENTIALS_SECRET_NAME} --set configSecret.namespace=${CREDENTIALS_SECRET_NAMESPACE}  --wait --timeout 90s
+```
+
+Docker provider can be replaced by any provider supported by [clusterctl](https://cluster-api.sigs.k8s.io/reference/providers.html?highlight=hetz#infrastructure)
+
+Other options for installing Cluster API Operator are described in [full documentation](README.md#installation).
+ 
+# Example API Usage
```

Comment:
This should not be an example API usage.  Technically people must configure the appropriate credentials for their provider for the provider to function. e.g. https://github.com/kubernetes-sigs/cluster-api-operator/blob/bd38f7a8b136e765351e524bfa50edf13c7b4683/docs/README.md#installing-azure-infrastructure-provider